applications/SolidMechanicsApplication/custom_utilities/solid_mechanics_math_utilities.hpp

```diff
@@ -1712,6 +1712,59 @@ class SolidMechanicsMathUtilities
             return true;
     }
 
+
+  inline static VectorType range(const unsigned int start,  const unsigned int end)
+  {
+    unsigned int size = end-start;
```

Comment:
I think you should consider removing the "unsigned". Somewhing weird may happen in the case that end-start is negative (also, the "size<0"  gives a compilation warning in my computer since size is unsigned)

Reply:
That's right